cmd/otelcol/config/collector/full_config_linux.yaml

```diff
@@ -167,6 +167,24 @@ receivers:
   fluentforward:
     endpoint: 0.0.0.0:8006
 
+  # Enables scripted inputs receiver
+  # Full configuration here: https://github.com/weliasz/splunk-otel-collector/tree/main/internal/receiver/scriptedinputsreceiver
+  scripted_inputs/cpu:
```

Comment:
I don't think the aim of this file is to demonstrate all possible receiver configurations (this would be very verbose for smart agent or filelog and processors, etc.) so would suggest just adding a single one.